sql-statements/sql-statement-overview.md

```diff
@@ -0,0 +1,326 @@
+| [`SHOW ENGINES`](/sql-statements/sql-statement-show-engines.md) | Shows available storage engines. |
+| [`SHOW PLUGINS`](/sql-statements/sql-statement-show-plugins.md) | Shows installed plugins. |
+| [`SHOW PROCESSLIST`](/sql-statements/sql-statement-show-processlist.md) | Shows the current sessions connected to the same TiDB server. |
+| [`SHOW PROFILES`](/sql-statements/sql-statement-show-profiles.md) | Shows query profiles. Included for [compatibility with MySQL](/mysql-compatibility.md). Currently only returns an empty result. |
+
+</CustomContent>
+
+## Locking statements
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`LOCK STATS`](/sql-statements/sql-statement-lock-stats.md) | Locks statistics of tables or partitions. |
+| [`LOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) | Locks tables for the current session. |
+| [`UNLOCK STATS`](/sql-statements/sql-statement-unlock-stats.md) | Unlocks statistics of tables or partitions. |
+| [`UNLOCK TABLES`](/sql-statements/sql-statement-lock-tables-and-unlock-tables.md) | Unlocks tables. |
+
+## Account management / Data Control Language
+
+| SQL Statement | Description |
+|---------------|-------------|
+| [`ALTER USER`](/sql-statements/sql-statement-alter-user.md) | Modifies a user. |
+| [`CREATE ROLE`](/sql-statements/sql-statement-create-role.md) | Creates a role. |
+| [`CREATE USER`](/sql-statements/sql-statement-create-user.md) | Creates a new user. |
+| [`DROP ROLE`](/sql-statements/sql-statement-drop-role.md) | Drops an existing role. |
+| [`DROP USER`](/sql-statements/sql-statement-drop-user.md) | Drops an existing user. |
+| [`FLUSH PRIVILEGES`](/sql-statements/sql-statement-flush-privileges.md) | Reloads the in-memory copy of privileges from the privilege tables. |
+| [`GRANT <privileges>`](/sql-statements/sql-statement-grant-privileges.md) | Grants privileges. |
+| [`GRANT <role>`](/sql-statements/sql-statement-grant-role.md) | Grants a role. |
+| [`RENAME USER`](/sql-statements/sql-statement-rename-user.md) | Renames an existing user. |
+| [`REVOKE <privileges>`](/sql-statements/sql-statement-revoke-privileges.md) | Revokes privileges. |
+| [`REVOKE <role>`](/sql-statements/sql-statement-revoke-role.md) | Revokes a role. |
+| [`SET DEFAULT ROLE`](/sql-statements/sql-statement-set-default-role.md) | Sets a default role. |
+| [`SET PASSWORD`](/sql-statements/sql-statement-set-password.md) | Changes a password. |
+| [`SET ROLE`](/sql-statements/sql-statement-set-role.md) | Enables roles in the current session. |
+| [`SHOW CREATE USER`](/sql-statements/sql-statement-show-create-user.md) | Shows the `CREATE` statement for a user. |
+| [`SHOW GRANTS`](/sql-statements/sql-statement-show-grants.md) | Shows privileges associated with a user. |
+| [`SHOW PRIVILEGES`](/sql-statements/sql-statement-show-privileges.md) | Shows available privileges. |
+
+## TiDB Binlog & TiCDC
```

Comment:
```suggestion
## TiCDC & TiDB Binlog
```

Reply:
According to https://docs.pingcap.com/tidb/stable/tidb-binlog-overview, "Starting from TiDB v7.5.0, technical support for the data replication feature of TiDB Binlog is no longer provided. It is strongly recommended to use [TiCDC](https://docs.pingcap.com/tidb/stable/ticdc-overview) as an alternative solution for data replication.", it is   better to highlight TiCDC first.